services/inspection-report/src/components/ResponsiveAppBar/index.js

```diff
@@ -51,7 +51,7 @@ export default function ResponsiveAppBar() {
   };
 
   return (
-    <AppBar position="static">
+    <AppBar position="static" style={{ zIndex: 999 }}>
```

Comment:
Not a good idea. AppBar can steel be in background of a modal backdrop for example.
We shouldn't modify z-index of MUI components.